### ISSUE TITLE
Uncaught TypeError: undefined is not a function (Detect conflicts)

### ISSUE THREAD

**Issue by maaarghk** [smashwilson/merge-conflicts#169]
I commented this on #168 but the submitter there pointed out quite rightly that the stack trace is totally different, so I've opened a separate issue.

1. Press ctrl m d or "Detect" from packages submenu.

**Atom Version**: 1.0.2
**System**: Mac OS X 10.10.4
**Thrown From**: [merge-conflicts](https://github.com/smashwilson/merge-conflicts) package, v1.3.3

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/mark/.atom/packages/merge-conflicts/lib/view/merge-conflicts-view.coffee:158

TypeError: undefined is not a function
  at Function.MergeConflictsView.detect (/Users/mark/.atom/packages/merge-conflicts/lib/view/merge-conflicts-view.coffee:158:22)
  at /Users/mark/.atom/packages/merge-conflicts/lib/main.coffee:27:28
  at /Users/mark/.atom/packages/merge-conflicts/lib/git-bridge.coffee:42:9
  at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:213:47)
  at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:235:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:169:7)
  at Process.ChildProcess._handle.onexit (child_process.js:1057:12)

```

### Commands

```
     -4:13.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -3:59.2.0 editor:newline (atom-text-editor.editor.is-focused)
 47x -3:55.6.0 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
     -2:54 core:move-right (atom-text-editor.editor.is-focused)
     -2:53.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -2:51.8.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -2:49.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -2:44.5.0 editor:newline (atom-text-editor.editor.is-focused)
     -2:39.9.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -2:39.9.0 snippets:expand (atom-text-editor.editor.is-focused)
     -2:39.9.0 editor:indent (atom-text-editor.editor.is-focused)
     -2:38.8.0 core:save (atom-text-editor.editor.is-focused)
     -1:22.9.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
     -1:21.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:04.7.0 application:minimize (atom-text-editor.editor.is-focused)
     -0:00.9.0 merge-conflicts:detect (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "unity-ui",
      "monokai"
    ]
  },
  "merge-conflicts": {}
}
```

### Installed Packages

```coffee
# User
SFTP-deployment, v1.0.0
atomatigit, v1.5.3
auto-indent, v0.5.0
language-apache, v1.3.0
language-ini, v1.11.0
merge-conflicts, v1.3.3
monokai, v0.15.0
project-manager, v1.15.11
unity-ui, v2.0.11

# Dev
No dev packages
```

**Comment by brodock** [comment]
same here

**Comment by radko93** [comment]
Same!

**Comment by lazutkin** [comment]
:+1:

**Comment by smashwilson** [comment]
Huh, this one's strange. Looking at [that line](https://github.com/smashwilson/merge-conflicts/blob/v1.3.3/lib/view/merge-conflicts-view.coffee#L158) from the trace, it's just calling a method on `GitBridge`... but `GitBridge` is already defined and imported properly. I also can't seem to reproduce. This is happening for *every* `merge-conflicts:detect` invocation?

I'll keep looking :mag_right: :footprints:

**Comment by smashwilson** [comment]
Oh, wait, "ctrl-M D"? 1.3.3 actually changes the keymap entries to "alt-M D" to avoid a conflict (#157).

Is there any chance that merge-conflicts has been updated, but Atom hasn't been relaunched? I'm wondering if a live update to 1.3.3 is leaving obsolete callbacks registered that reference dead code.

**Comment by maaarghk** [comment]
That's certainly possible. I will relaunch and see if I can reproduce after.

**Comment by kcaulkins** [comment]
I'm getting this when trying to press "Quit" after fixing conflicts

**Comment by maaarghk** [comment]
Relaunching atom solves issue for me.

**Comment by brunoric** [comment]
Same here.

**Comment by tvandervorm** [comment]
Got this error when pressing the 'stage' button after I'd already staged the changes using the commandline. Recent install of atom (< 2 weeks old), installed merge-conflicts today.

**Comment by tvandervorm** [comment]
Got the same error after pressing the 'stage' button while I'd already staged using the command line. Trace is slightly different but my guess is it's the same issue. Installed merge-conflicts today, so should be latest version, running on linux (Fedora).

    TypeError: undefined is not a function
      at Function.GitBridge.add (/home/tvandervorm/.atom/packages/merge-conflicts/lib/git-bridge.coffee:194:28)
      at MergeConflictsView.stageFile (/home/tvandervorm/.atom/packages/merge-conflicts/lib/view/merge-conflicts-view.coffee:147:15)
      at HTMLButtonElement.<anonymous> (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:220:36)
      at HTMLButtonElement.jQuery.event.dispatch (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
      at HTMLButtonElement.elemData.handle (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

**Comment by vasyan** [comment]
Me too. After click on 'stage'. OSX.

```
TypeError: undefined is not a function
  at Function.GitBridge.add (/Users/vasyan/.atom/packages/merge-conflicts/lib/git-bridge.coffee:194:28)
  at MergeConflictsView.stageFile (/Users/vasyan/.atom/packages/merge-conflicts/lib/view/merge-conflicts-view.coffee:147:15)
  at HTMLButtonElement.<anonymous> (/Users/vasyan/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:220:36)
  at HTMLButtonElement.jQuery.event.dispatch (/Users/vasyan/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLButtonElement.elemData.handle (/Users/vasyan/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
```

**Comment by smashwilson** [comment]
@tvandervorm, @vasyan: Ah! That's a different bug, but a pretty easy one. (I missed a call site in one of my refactorings.)

**Comment by smashwilson** [comment]
@tvandervorm, @vasyan: Replicated locally, fixed in #173, and released as 1.3.5. :sparkles:
